config/site/src/guides.md

```diff
@@ -6,6 +6,16 @@ permalink: /guides/
 
 Learn how to build and scale your ElasticGraph applications with our comprehensive guides.
 
-- [Getting Started]({% link getting-started.md %})
+## Available Guides
 
-- [AI Tools]({% link guides/ai-tools.md %})
+{% assign guides = site.pages | where_exp: "item", "item.path contains 'guides/'" | sort: "title" %}
+{% for guide in guides %}
+- [{{ guide.title | default: guide.name | remove: '.md' }}]({{ guide.url | relative_url }})
+{%- if guide.description %}
+  - {{ guide.description }}
```

Comment:
Do we have any guides with descriptions to exercise this? (Should we add one to `AI Tools`)? 

Also do you think the description is better a sub-bullet or as `Title: Description`?

Or we could use a definition list, similar to the API predicates:

https://block.github.io/elasticgraph/query-api/filtering/available-predicates/

(That would be especially great if the description might be long).